suite-native/module-connect-device/src/screens/ConnectingDeviceScreen.tsx

```diff
@@ -15,7 +15,7 @@ import { ConnectDeviceSreenView } from '../components/ConnectDeviceSreenView';
 const screenStyle = prepareNativeStyle(() => ({
     justifyContent: 'center',
     alignItems: 'center',
-    paddingBottom: 80,
+    paddingBottom: 150,
```

Comment:
I still think this is not right

Reply:
Can you please show screenshot from iPad?

Reply:
without paddingBottom:
![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2023-12-07 at 21 02 30](https://github.com/trezor/trezor-suite/assets/2011829/807d9707-a145-4ac4-8c8e-f6af3bc700eb)

with paddingBottom: 150:
![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2023-12-07 at 21 02 35](https://github.com/trezor/trezor-suite/assets/2011829/5d3f2ebc-9a8d-4b03-91fd-af65e02e83cf)